### PR TITLE
Fixes #20020: parse parameter_rename in methods metadata

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/20020

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
Commit modified

File: tools/ncf.py [hunk 1]
```diff
@@ -194,6 +195,10 @@ def parse_bundlefile_metadata(content, bundle_type):
         if tag == "parameter_type":
           param_type = match.group(4)
           param_types[match.group(3)] = param_type
+        if tag == "parameter_rename":
+          new_name = match.group(3)
+          old_name = match.group(4)
```
[the remark this answers is not in the document]
Comment (VinceMacBuche) [reply tools/ncf.py]: I thought i did that, maybe only done this in my dreams

**Review by peckpeck — APPROVED** [review]

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
OK, merging this PR

**Pull request merged**